src/NLog/Targets/WebServiceTarget.cs

```diff
@@ -733,35 +728,13 @@ protected HttpPostTextFormatterBase(WebServiceTarget target) : base(target)
                 _encodingPreamble = target.Encoding.GetPreamble();
             }
 
-            protected abstract string Separator { get; }
-
-            protected virtual void BeginFormattedMessage(StringBuilder builder)
-            {
-            }
-
-            protected abstract void AppendFormattedParameter(StringBuilder builder, MethodCallParameter parameter, object value);
```

Comment:
is this a breaking changes? (removing the abstract methods)

Reply:
Nope those are private classes inside WebServiceTarget